RoktDemo/Model/PreDefinedScreen1Model.swift

```diff
@@ -0,0 +1,18 @@
+//
```

Comment:
Is there a difference between PredefinedScreenModel 1,2 and 3? Can they all be the same class?